<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>223</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>223</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>223</width>
     <height>64</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/pictures/main interface/start_deep.png);
}
QPushButton:checked{
	border-image: url(:/pictures/main interface/start.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>223</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>223</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>223</width>
     <height>64</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/pictures/main interface/stop_deep.png);
}
QPushButton:checked{
	border-image: url(:/pictures/main interface/stop.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>155</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>155</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>155</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MINI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>765</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>765</width>
     <height>33</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>756</width>
     <height>33</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/MINI.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rotation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>147</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>147</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pictures/main interface/rotation.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ground_bg1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>194</width>
     <height>288</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ground_bg2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>194</width>
     <height>288</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ground_bg3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>194</width>
     <height>288</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>288</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rpm1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>29</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_rpm.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rpm2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>29</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_rpm.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rpm3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>210</y>
     <width>29</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group_rpm.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pictures/main interface/group3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	qproperty-alignment:AlignCenter;
	border-image: url(:/pictures/main interface/Input_box.png);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	qproperty-alignment:AlignCenter;
	border-image: url(:/pictures/main interface/Input_box.png);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>142</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	qproperty-alignment:AlignCenter;
	border-image: url(:/pictures/main interface/Input_box.png);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
